common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryEventStrings.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.telemetry;
+
+public final class TelemetryEventStrings {
+    public static final String TELEMETRY_EVENT_DEFAULT_EVENT = "default_event";
+
+    public static final String TELEMETRY_EVENT_API_EVENT_START = "api_start_event";
+    public static final String TELEMETRY_EVENT_API_EVENT_END = "api_end_event";
+
+    public static final String TELEMETRY_EVENT_CACHE_EVENT_START = "cache_start_event";
+    public static final String TELEMETRY_EVENT_CACHE_EVENT_END = "cache_end_event";
+
+    public static final String TELEMETRY_EVENT_UI_EVENT_START = "ui_start_event";
+    public static final String TELEMETRY_EVENT_UI_EVENT_END = "ui_end_event";
+
+    public static final String TELEMETRY_EVENT_HTTP_REQUEST_START = "http_start_event";
```

Comment:
can these be changed to match the constants name for consistency ?  like "http_start_event" - > "http_request_start_event"

Feedback applies for the constants below